examples/js/loaders/GLTF2Loader.js

```diff
@@ -1668,10 +1693,12 @@ THREE.GLTF2Loader = ( function () {
 	GLTFParser.prototype.loadMeshes = function () {
 
 		var json = this.json;
+		var extensions = this.extensions;
 
 		return this._withDependencies( [
 
 			"accessors",
```

Comment:
In loadAccessors function, is it required to check if the `accessor` has `bufferView` before this line:
`var arraybuffer = dependencies.bufferViews[ accessor.bufferView ];`